### ISSUE TITLE
You get get flagged at war with whatever neutral town your in if you attack other faction

### ISSUE THREAD

**Issue by Roflkopterr** [Wall-core/Everlook-Bugtracker#358]
This is probably a Vmangos issue as it happened on Hyjal AND darrow. No idea how this has not been fixed or reported after THIS long.

You should not get flagged at war just for pvping in a neutral town. That was never a thing in any version of the game.

**Comment by Wall-core** [comment]
This happened for sure in original wrath, will have to confirm for vanilla tho.

**Comment by Roflkopterr** [comment]
Yes thats the problem. I am guessing thats why it was added to Vmangoes since they pulled a lot of data from 1.14 and got shit from expansions after vanilla.

This 100% did not occur on vanilla classic.
